pyprusalink/types.py

```diff
@@ -0,0 +1,64 @@
+from typing import TypedDict
+
+from .const import API_KEY_AUTH, DIGEST_AUTH
+
+
+class VersionInfo(TypedDict):
```

Comment:
Since there are multiple versions of the API that we all want to support, we should probably include a version number in the type names.